packages/core/micro/src/adjust-tree/types.ts

```diff
@@ -30,7 +30,7 @@ export interface SegmentRange<T> {
 export interface TreeConfiguration<T> {
     readonly emptySegment: T;
     readonly order: number;
-    readonly deleteSegmentRange: (segment: T, start: number, count: number) => [T, T];
+    readonly deleteSegmentRange: (segment: T, start: number, length: number) => [T, T];
```

Comment:
Just FYI - Every time I've benchmarked it, it's been faster to return an object literal like this:

const { left, right } = tree.deleteSegmentRange(..);

than an array like this:

const [ left, right ] = tree.deleteSegmentRange(..);

...which is a bummer, because after array tuples are more natural.

Reply:
(...just passing along.  no need to change.)

Reply:
I shall change this, I assumed the tuple was faster! I prefer the object